### PR TITLE
Create Jenkins pipelines

### PR DESCRIPTION
**Pull request by olavph**

### CODE REVIEW

File: jenkins_jobs/build_pipelines.groovy [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        buildForkPRHead(true)
+        buildForkPRMerge(false)
+        buildOriginBranch(false)
+        buildOriginBranchWithPR(false)
+        buildOriginPRHead(true)
+        buildOriginPRMerge(false)
+      }
+    }
+    triggers {
+      periodic(1)
```
Comment (farosas) [inline jenkins_jobs/build_pipelines.groovy:17]: This has a periodic trigger but `createPeriodicPipeline` don't, is that correct?

File: jenkins_jobs/periodic_pipelines.groovy [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  dir('infrastructure') {
+    git(url: 'ssh://git@github.com/$GITHUB_ORGANIZATION_NAME/infrastructure.git',
+        branch: 'pipeline')
+  }
+  pipeline = load 'infrastructure/pipeline/${name}.groovy'
+}
+
+pipeline.execute()
+""")
+        sandbox()
```
Comment (farosas) [inline jenkins_jobs/periodic_pipelines.groovy:16]: What does this do?

File: jenkins_jobs/build_pipelines.groovy (hunk 1 above)
Comment (olavph) [reply jenkins_jobs/build_pipelines.groovy:17]: Yes. This trigger is for the polling of changes in the repositories.

File: jenkins_jobs/periodic_pipelines.groovy (hunk 2 above)
Comment (olavph) [reply jenkins_jobs/periodic_pipelines.groovy:16]: It allows the execution of the script, as long as the methods used are approved.
Without this option, an admin would have to manually approve the whole script each time it changes.
https://support.cloudbees.com/hc/en-us/articles/207406207-Avoid-script-approvals-with-a-Jenkins-Pipeline-Groovy-script

**Review by farosas — APPROVED** [review]

**Pull request merged**